### ISSUE TITLE
Deprecation warning in htmlspecialchars() in DashboardPanel.class.php

### ISSUE THREAD

**Issue by donatasben** [philippdaun/processwire-dashboard#54]
Hi,
I am getting this deprecation warning
```
Deprecated: htmlspecialchars(): Passing null to parameter #1 ($string) of type string is deprecated in 
.../site/modules/Dashboard/DashboardPanel.class.php:414
```
in
https://github.com/daun/processwire-dashboard/blob/b4436b7f4eed7ffed0f76ebc9f39a2a117f5b372/DashboardPanel.class.php#L414

However it was my Dashboards configuration that causes this - I skipped the "`parent`" param for "**PageList**" panel type. I wanted to show the default full page list, so skipped the "`parent`" param as it said that default is `Homepage` in your [docs](https://daun.github.io/processwire-dashboard/#/panels/page-list?id=panel-type-pagelist). I assumed I can skip it if default is the one I want.
If I set `'parent' => 'id=1'` - the error goes away.

Suggestions: it should set the default Homepage ID to $attributes param array before running renderAttributes() function.
Or modify renderAttributes() function to skip the attribute `if ($val === null)`.
Or update the docs to tell it's a required parameter.

**Comment by daun** [comment]
Good to know you found a fix, but I'll need to check the module for compatibility with PHP 8.4 soon anyway. I'll make a mental note to set a default value here as well.

**Comment by daun** [comment]
I've added a defensive check for null values in the mentioned method. Released as [1.5.7](https://github.com/daun/processwire-dashboard/releases/tag/v1.5.7). Let me know if you're still seeing the issue and thanks for reporting :)
